src/generators/indexGenerator.ts

```diff
@@ -33,4 +34,32 @@ export function generateIndexFile(
       namedExports: [`${clientDetails.className}Context`]
     }
   ]);
+
+  let includeLRO = false;
+  const operationGroups = clientDetails.operationGroups.filter(
+    og => !og.isTopLevel
+  );
+
+  if (operationGroups.length) {
+    indexFile.addExportDeclarations([
+      {
+        moduleSpecifier: "./operationsInterfaces"
+      }
+    ]);
+  }
+
+  operationGroups.forEach(og => {
+    if (hasLROOperation(og)) {
+      includeLRO = true;
+    }
+  });
```

Comment:
NIT: `includeLRO` could be defined in one line as follows:
```suggestion
  const operationGroups = clientDetails.operationGroups.filter(
    og => !og.isTopLevel
  );

  if (operationGroups.length) {
    indexFile.addExportDeclarations([
      {
        moduleSpecifier: "./operationsInterfaces"
      }
    ]);
  }

  const includeLRO = operationGroups.find(hasLROOperation) !== undefined;
```